contracts/Bridge.sol

```diff
@@ -3,8 +3,10 @@ pragma solidity 0.8.11;
 
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
 import "@openzeppelin/contracts/utils/Context.sol";
+import "@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol";
```

Comment:
They have not-draft version. Looks the same though 
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/cryptography/EIP712.sol

Reply:
We're using `@openzeppelin/contracts` v4.5.0 which  doesn't have `"@openzeppelin/contracts/utils/cryptography/EIP712.sol";`, I would rather leave it for now as is.